src/windows_clipboard.rs

```diff
@@ -19,16 +19,16 @@ use clipboard_win::Clipboard;
 use common::ClipboardProvider;
 use std::error::Error;
 
-pub struct WindowsClipboardContext(Clipboard);
+pub struct WindowsClipboardContext;
 
 impl ClipboardProvider for WindowsClipboardContext {
     fn new() -> Result<Self, Box<Error>> {
-        Ok(WindowsClipboardContext(Clipboard::new()?))
+        Ok(WindowsClipboardContext)
     }
     fn get_contents(&mut self) -> Result<String, Box<Error>> {
-        Ok(self.0.get_string()?)
+        Ok(Clipboard::new()?.get_string()?)
```

Comment:
I suggest to use shortcut methods `get_clipboard_string` and `set_clipboard_string`